server/src/main/java/org/eclipse/openvsx/admin/AdminService.java

```diff
@@ -275,6 +275,7 @@ public UserPublishInfoJson getUserPublishInfo(String provider, String loginName)
                     var json = latest.toExtensionJson();
                     json.preview = latest.isPreview();
                     json.active = latest.getExtension().isActive();
+                    json.potentiallyMalicious = latest.isPotentiallyMalicious();
```

Comment:
This is going to be used in the webui?

Reply:
Not planned yet, therefore I removed it again. We can add it back if there ever is a plan to show it in the webui.